plugin/builtin/legacyordertransformer/Makefile

```diff
@@ -1 +0,0 @@
-../../../Makefile-modules.mk
```

Comment:
Please add a makefile symlink like this in the new directory so that it can be picked up by our linter/tests/other various tasks.